packages/protocol/test/governance/release_gold.ts

```diff
@@ -25,9 +25,8 @@ import {
   MockValidatorsInstance,
```

Comment:
Summary of changes:
Name changes and removing factory code (tests and utilities).